lib/measured/measurable.rb

```diff
@@ -53,9 +53,7 @@ def <=>(other)
 
   class << self
 
-    def conversion
-      @conversion ||= Measured::Conversion.new
-    end
+    attr_reader :conversion
```

Comment:
You could redefine this:

```ruby
def conversion
  raise Measured::UnitError, "Conversion not defined" unless @conversion
  @conversion
end
```

Down side is that we would then fire this check on _every_ type of conversion. Might be a better place for this...